resolwe_bio/processes/alignment/meta.yml

```diff
@@ -0,0 +1,26 @@
+# =================================
+# Parent of all alignemnt processes
+# =================================
+
+- slug: meta-alignment
+  name: Meta alignment process
+  version: 1.0.0
+  type: data:alignment
```

Comment:
Should we add this processes to a dedicated category, say meta or abstract? This will make it easier to exclude them from the UI. @dblenkus How should we name the category?

Reply:
I would prefer abstract, but somehow meta just stuck in my head during conversations. I'm open to better suggestions...

Reply:
Let's call this abstract as this is more usual for this type of concepts in CS terminology. So, change the slugs to abstract- and add a category.